bigquery/integration_test.go

```diff
@@ -889,6 +899,88 @@ func TestIntegration_Tables(t *testing.T) {
 	}
 }
 
+// setupPolicyTag is a helper for setting up policy tags in the datacatalog service.
+//
+// It returns a string for a policy tag identifier and a cleanup function, or an error.
+func setupPolicyTag(ctx context.Context) (string, func(), error) {
+	location := "us"
+	req := &datacatalogpb.CreateTaxonomyRequest{
+		Parent: fmt.Sprintf("projects/%s/locations/%s", testutil.ProjID(), location),
+		Taxonomy: &datacatalogpb.Taxonomy{
```

Comment:
What's the resource identifier for a taxonomy? Will it assign a random ID? Do we have a way of cleaning these up if cleanupFunc fails to run?

Reply:
Yeah, identifier uses an integer value for the last token replacement in `projects/{project}/locations/{location}/taxonomies/{taxonomies}`.

Part of the reason I populate displayname and description is so these are easier to filter for if we need to do bulk cleanups.